SDWebImage/SDWebImageDownloader.h

```diff
@@ -91,6 +91,8 @@ FOUNDATION_EXPORT NSString * _Nonnull const SDWebImageDownloadStopNotification;
 typedef SDImageLoaderProgressBlock SDWebImageDownloaderProgressBlock;
 typedef SDImageLoaderCompletedBlock SDWebImageDownloaderCompletedBlock;
 
+@protocol SDWebImageDownloaderOperation;
```

Comment:
Emm.....What does this used for ? The `SDWebImageDownloadToken` just use a internal class category, don't need to change anything on the header files